CHANGELOG.md

```diff
@@ -4,6 +4,19 @@
 - Debug from .csproj and .sln [#5876](https://github.com/dotnet/vscode-csharp/issues/5876)
 
 # Latest
+* Update Razor to 9.0.0-preview.24467.1
+  * Fixing WrapWithDiv bug (PR: [#10871](https://github.com/dotnet/razor/pull/10871))
+  * Switch to new VMR control schema. (PR: [#10883](https://github.com/dotnet/razor/pull/10883))
```

Comment:
Consider removing any changes that aren't necessarily relevant to a vscode issue from the changelog

Reply:
+1, if you need help feel free to ping me and I can help reduce this list